artemis-cli/src/main/java/org/apache/activemq/artemis/cli/commands/Create.java

```diff
@@ -298,6 +298,9 @@ public String[] getStaticNodes() {
    @Option(names = "--security-manager", description = "Which security manager to use - jaas or basic. Default: jaas.")
    private String securityManager = "jaas";
 
+   @Option(names = "--journal-max-io", description = "The journal-max-io value to use when also using the ASYNCIO journal-type. When using NIO or MAPPED this value is always '1'. Default: 4096")
+   private long journalMaxIo = ActiveMQDefaultConfiguration.getDefaultJournalMaxIoAio();
```

Comment:
I just twigged with your most recent updates that this is a long...which on reflection seemed a bit large given the default, and this option being added because that is actually _too big_ in some envs.

On checking, journal-max-io is also defined as an _int_ for all the rest of the configuration bits, so this should match.

Reply:
Fixed.